recyclerviewfastscroller/src/main/java/com/qtalk/recyclerviewfastscroller/RecyclerViewFastScroller.kt

```diff
@@ -126,6 +127,7 @@ class RecyclerViewFastScroller @JvmOverloads constructor(
         const val DEFAULT_ANIM_DURATION: Long = 100
         const val DEFAULT_POPUP_VISIBILITY_DURATION = 200L
         const val hasEmptyItemDecorator: Boolean = true
+        const val hideHandleAfter: Int = 0
```

Comment:
let's change this as well and use uppercase for constant values 
```suggestion
        const val DEFAULT_HANDLE_VISIBILITY_DURATION: Int = 0
```

Reply:
I thought lowercase values are for the default property values. I think it's a different issue and handled separately.